.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: ruby
+dist: xenial
+rvm: 2.7.2
+bundler_args: --jobs=2
```

Comment:
I think this line runs bundle so the bundle command isn't needed under the script key

Reply:
You should be able to confirm the behavior this key in the TravisCI docs